src/components/Layout/Header/Header.tsx

```diff
@@ -43,6 +46,8 @@ export const Header = ({ route }: { route: Route }) => {
     return () => document.removeEventListener('keydown', handleKeyPress)
   }, [handleKeyPress])
 
+  const gemRampFlag = useSelector(state => selectFeatureFlag(state as any, 'GemRamp'))
```

Comment:
```suggestion
  const gemRampFlag = useAppSelector(state => selectFeatureFlag(state, 'GemRamp'))
```

Reply:
I initially went with this, but we get this bad boy error:
```
portfolioSlice.ts:170 Uncaught ReferenceError: Cannot access 'selectAccountIdByAddress' before initialization
```

I grepped `selectAccountIdByAddress` and couldn't find what caused it. `useAppSelector` should simply be a RTK-typed alias for `useSelector` so this one is weird. Can you reproduce it?